buildspecs/linux_aarch64.spec

```diff
@@ -0,0 +1,246 @@
+		<property name="graph_req.aux0" value=""/>
+		<property name="graph_req.build" value="{$common.req.build.java9$}"/>
+		<property name="graph_req.build2" value="{$common.req.build.java8$}"/>
+		<property name="graph_req.machine" value="{$machine_mapping.x86$}"/>
+		<property name="graph_req.os" value="{$machine_mapping.linux$}"/>
+		<property name="graph_req.os.build" value="{$spec.property.graph_req.os$}"/>
+		<property name="graph_req.os.perf" value="{$spec.property.graph_req.os$}"/>
+		<property name="graph_se_classlib.java5" value="jcl_se.zip"/>
+		<property name="graph_se_classlib.java6" value="jcl_se.zip"/>
+		<property name="graph_variants" value="linux_2.4,linux_2.6,linux_hammer"/>
+		<property name="isReallyUnix" value="true"/>
+		<property name="j2seRuntimeDir" value="jre/lib/aarch64"/>
+		<property name="j2seTags" value="pxr6460,j9vmxr6424"/>
+		<property name="j9BuildName" value="linux_aarch64"/>
+		<property name="j9dt.compileTarget" value="makefile"/>
+		<property name="j9dt.make" value="gmake"/>
+		<property name="j9dt.toolsTarget" value="buildtools.mk"/>
+		<property name="javatestPlatform" value="linux_aarch64"/>
+		<property name="jclMemoryMax" value="-Xmx32m"/>
+		<property name="jclOSStackSizeMax" value=""/>
+		<property name="jdiTestsSupported" value="true"/>
+		<property name="jgrinderTestingSupported" value="true"/>
+		<property name="jitTestingOptLevel" value="optlevel=warm"/>
+		<property name="localRootPath" value="$(J9_UNIX_ROOT)"/>
+		<property name="longLimitCmd" value=""/>
+		<property name="main_shortname" value="xr64"/>
+		<property name="os.lineDelimiter" value="unix"/>
+		<property name="platform_arch" value="arm"/>
+		<property name="sun.jdk7.platform_id" value="linux-aarch64"/>
+		<property name="sun.jdk8.platform_id" value="linux-aarch64"/>
+		<property name="svn_stream" value=""/>
+		<property name="uma_make_cmd_ar" value="$(OPENJ9_CC_PREFIX)-ar"/>
+		<property name="uma_make_cmd_as" value="$(OPENJ9_CC_PREFIX)-as"/>
+		<property name="uma_make_cmd_cc" value="$(OPENJ9_CC_PREFIX)-gcc"/>
+		<property name="uma_make_cmd_cpp" value="$(OPENJ9_CC_PREFIX)-cpp -E -P"/>
+		<property name="uma_make_cmd_cxx" value="$(OPENJ9_CC_PREFIX)-g++"/>
+		<property name="uma_make_cmd_cxx_dll_ld" value="$(CC)"/>
+		<property name="uma_make_cmd_cxx_exe_ld" value="$(CC)"/>
+		<property name="uma_make_cmd_dll_ld" value="$(CC)"/>
+		<property name="uma_make_cmd_exe_ld" value="$(CC)"/>
+		<property name="uma_make_cmd_ranlib" value="$(OPENJ9_CC_PREFIX)-ranlib"/>
+		<property name="uma_processor" value="aarch64"/>
+		<property name="uma_type" value="unix,linux"/>
+	</properties>
+	<features>
+		<feature id="combogc"/>
+		<feature id="core"/>
+		<feature id="dbgext"/>
+		<feature id="se"/>
+		<feature id="se60_26"/>
+		<feature id="se7"/>
+		<feature id="se70_27"/>
+	</features>
+	<source>
+		<project id="com.ibm.jvmti.tests"/>
+		<project id="compiler"/>
+	</source>
+	<flags>
+		<flag id="arch_arm" value="true"/>
```

Comment:
This shouldn't be `true` for aarch64

Reply:
Thank you for pointing it out.  I corrected it, and the `j9vm_arch_*=`section in makelib/mkconstants.mk is now OK.